pipeline/tests/powdr_std.rs

```diff
@@ -165,27 +165,25 @@ fn permutation_via_challenges_ext() {
 }
 
 #[test]
-fn lookup_via_challenges_bn() {
+fn lookup_via_challenges() {
     let f = "std/lookup_via_challenges.asm";
     test_halo2(make_simple_prepared_pipeline(f));
-}
 
-#[test]
-fn lookup_via_challenges_ext() {
-    let f = "std/lookup_via_challenges_ext.asm";
-    test_halo2(make_simple_prepared_pipeline(f));
-    // Note that this does not actually run the second-phase witness generation, because no
-    // Goldilocks backend support challenges yet.
+    // TODO: Plonky3 fails, not sure why...
```

Comment:
Did you find out more about this?

Reply:
That should be fixed by #1967, which is not in this branch yet, but will be after re-basing to `main`!

Reply:
ooooooook